packages/components/psammead-oembed/src/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { string, shape } from 'prop-types';
+import DOMPurify from 'dompurify';
+
+const DOMPURIFY_CONFIG = {
+  ADD_TAGS: ['iframe'],
+};
+
+const OEmbed = ({ oEmbed }) => {
+  const { html } = oEmbed;
+  const sanitizedHtml = DOMPurify.sanitize(html, DOMPURIFY_CONFIG);
```

Comment:
Is this sanitiser going to strip out the `<script>` at the end of the HTML in the Instagram [fixture](https://github.com/bbc/psammead/blob/50f9dd00cb92e0fa3d94cad2b06eb5ef8b3d10d7/packages/components/psammead-oembed/src/fixtures/instagram.json#L13)?

Reply:
It will, yes!